site/gatsby-site/src/templates/landingPage.js

```diff
@@ -45,9 +46,7 @@ const LandingPage = (props) => {
   const { locale: language } = useLocalization();
 
   const latestReports = latestReportIncidents.edges.map((incident) => {
-    const sortedReports = sortBy(incident.node.reports, ['epoch_date_submitted'], ['desc']);
-
-    const report = sortedReports.map((report) => report)[0];
```

Comment:
This report was not included in here:

https://github.com/responsible-ai-collaborative/aiid/blob/0ba916477241d29f7aed9c91741801e9c79f915d/site/gatsby-site/page-creators/createWordCountsPage.js#L128-L134

So it wasn't being included here:

https://github.com/responsible-ai-collaborative/aiid/blob/0ba916477241d29f7aed9c91741801e9c79f915d/site/gatsby-site/src/templates/landingPage.js#L233-L243

Reply:
Goodness. Such a weird way for the report to evade translation... Thanks for digging into this.

Reply:
Yeah it is not pretty